src/main/java/com/epam/ta/reportportal/dao/util/RecordMappers.java

```diff
@@ -117,4 +138,63 @@
 			.collect(Collectors.toMap(r -> r.get(LAUNCH.ID), r -> r.into(Launch.class)))
 			.values());
 
+	/**
+	 * Maps record into {@link User} object
+	 */
+	public static final RecordMapper<? super Record, User> USER_RECORD_MAPPER = r -> {
+		User user = new User();
+		Project defaultProject = new Project();
+		String metaDataString = r.get(fieldName(USERS.METADATA), String.class);
+		ofNullable(metaDataString).ifPresent(md -> {
+			try {
+				JsonbObject metaData = new ObjectMapper().readValue(metaDataString, JsonbObject.class);
```

Comment:
Do we have object mapper into spring context or not? Do wer need to create an instance of object mapper every time?